br2_external/configs/X6100_defconfig

```diff
@@ -2,7 +2,15 @@ BR2_arm=y
 BR2_cortex_a7=y
 BR2_ARM_FPU_NEON_VFPV4=y
 BR2_TOOLCHAIN_EXTERNAL=y
-BR2_TOOLCHAIN_EXTERNAL_LINARO_ARM=y
+BR2_TOOLCHAIN_EXTERNAL_CUSTOM=y
+BR2_TOOLCHAIN_EXTERNAL_DOWNLOAD=y
+BR2_TOOLCHAIN_EXTERNAL_URL="https://snapshots.linaro.org/gnu-toolchain/12.2-2022.12-1/arm-linux-gnueabihf/gcc-linaro-12.2.1-2022.12-x86_64_arm-linux-gnueabihf.tar.xz"
+BR2_TOOLCHAIN_EXTERNAL_CUSTOM_PREFIX="arm-linux-gnueabihf"
+BR2_TOOLCHAIN_EXTERNAL_HEADERS_5_15=y
```

Comment:
I don't understand this line... Why `HEADERS_5_15=y`?

Reply:
This is the maximum that the toolchain knows about. It eventually still uses from the kernel sources